aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -875,7 +875,7 @@ private SerializedCustomType createSerializedCustomType(
                 continue;
             }
 
-            if (field.isCustomType()) {
+            if (field.isCustomType() && entry.getValue() != null) {
```

Comment:
What is the field was defined as `nullable`? Shouldn't the serialized custom type still have the field included in that case?

Reply:
Ahh yes, thanks for catching it. Will update.

Reply:
Thanks for looking!
If `field.isCustomType() && entry.getValue() != null` is false it should fallback to line 895, which should be able to handle the nullable field case.

Reply:
I must have missed that :)